PathOfBuildingCommunity/PathOfBuilding
# Refactor project structure

Preparatory changes for automated testing and version 2.0.0:
* Change project structure to `src` layout
* Unpack runtime to `runtime` directory
* Update launcher to latest version to recognise new layout
* Delete skill tree zip files as they are no longer needed
* Add new manifest sources to support new layout
* Update manifest updater script to recognise new layout
* Update documentation to reflect new layout

Note that even though future versions of Path of Building will update with the files in the `runtime` directory, unlike the skill tree zip files, `runtime-win32.zip` has to be kept and maintained indefinitely, for backwards compatibility. Otherwise, updating from installations <= 1.4.170.27 would fail to update `Path of Building.exe` due to the whitespace characters in that file name. Newer versions of Path of Building are patched and don't exhibit this behaviour anymore. However, it is not possible to patch older versions to perform an intermediate update to patch the downloader, before attempting to download the latest version.
